TwitchPlaysAssembly/Src/Commands/BombCommands.cs

```diff
@@ -108,6 +108,59 @@ public static IEnumerator Explode(TwitchBomb bomb, string user, bool isWhisper)
 	[Command(@"(explode|detonate|kapow)", AccessLevel.Mod, AccessLevel.Mod)]
 	public static IEnumerator Explode(TwitchBomb bomb) => bomb.DelayBombExplosionCoroutine();
 
+	#region Voting
+	/// <name>Start a vote</name>
+	/// <syntax>vote [action]</syntax>
+	/// <summary>Starts a vote about doing an action</summary>
+	[Command(@"vote (explode|detonate|kapow)")]
+	public static void VoteStart(TwitchBomb bomb, string user, [Group(1)] bool Detonation) => Votes.StartVote(bomb, user, Detonation ? "Detonation" : "");
```

Comment:
```suggestion
	public static void VoteStart(TwitchBomb bomb, string user) => Votes.StartVote(bomb, user, "Detonation");
```
The `Detonation` variable will never be false since that group is required.

Reply:
Yeah it’ll affect the command when there will be new stuff to vote on. And then you only need to add a new group and a new boolean